src/renderer/components/NewProjectModal/RunTranscriptionView.tsx

```diff
@@ -91,22 +83,69 @@ const RunTranscriptionView = ({ closeModal, nextView }: Props) => {
     return null;
   }
 
+  const getFileName = async () => {
+    const name = await extractFileNameWithExtension(
+      currentProject.mediaFilePath
+    );
+    setMediaFileName(name);
+  };
+
+  const projectName = currentProject.name;
+  getFileName();
+
+  const completedButton = (
+    <CustomButton
+      color="primary"
+      onClick={nextView}
+      disabled={asyncState !== AsyncState.DONE}
+      sx={{ width: '100%' }}
+    >
+      Get Started
+    </CustomButton>
+  );
+
   return (
-    <>
-      <Typography fontWeight="bold">{currentProject.name}</Typography>
-      <Typography>Please wait while we transcribe your video</Typography>
-      {asyncState}
-      <pre>{JSON.stringify(currentProject)}</pre>
-      <ButtonContainer>
-        <CancelButton onClick={closeModal}>Cancel</CancelButton>
-        <ActionButton
-          onClick={nextView}
-          disabled={asyncState !== AsyncState.DONE}
-        >
-          Done
-        </ActionButton>
-      </ButtonContainer>
-    </>
+    <Container sx={{ height: { xs: 500 } }}>
```

Comment:
@emmapainter Do we want the modal to be 500px tall when the user is on a small screen?

Reply:
Update: In this commit https://github.com/patrickbrett/mlvet/pull/51/commits/5af1389d001b5dd28148173ef8076759f31eff30 the height was updated to 400